doc/WELOTEC.md

```diff
@@ -0,0 +1,21 @@
+# Arrakis-pico-MK3
+
+- **Product Page:** https://www.welotec.com/de/industrie-pcs/
+- **BSP (dynamic layer):** ToDo
+- **Boot Media:**
+  - 1st it is tried to boot *omnect-os* from USB media 
+  - 2nd if USB media is not present the internal nvme will be booted.
+- **Flash internal nvme:**
+  - boot *omnect-os* from USB
+  - use [flash-mode 1](../README.md#flash-mode-1) in order to flash an image from USB to nvme. The nvme device path is */dev/nvme0n1*
+  - remove USB media and power on device 
+
```

Comment:
in other board/platform documents follows UART (/dev/ttyS0) information here.
is it intentionally left out here?

Reply:
ok, also with arrakis device the serial console is freely usable, its only us who use it for test purposes and (want to) inject a suitable configuration.